django_slack/utils.py

```diff
@@ -10,14 +10,14 @@ class Backend(object):
     def send(self, url, data):
         raise NotImplementedError()
 
-    def validate(self, content_type, content):
+    def validate(self, content_type, content, original_data):
```

Comment:
If we're considering renaming it everywhere, then I wonder if `message_data` is a better name? This still leaves `validate` with a slightly less great name, but I think the overall improvement is probably a better trade-off.

Thoughts?